docs/examples/metrics_all.json

```diff
@@ -30,8 +30,8 @@
     "dt_builtin_count_incl": 2,
     "dt_notbuiltin_count": 0,
     "dt_notbuiltin_count_incl": 0,
-    "expressivity": "SRIQ",
-    "expressivity_incl": "SRIQ",
+    "expressivity": "UCINTERIQ",
```

Comment:
What on earth is `UCINETERIQ`???? cc @ignazio1977

Reply:
Probably coming from [here](https://github.com/owlcs/owlapi/blob/owlapi-parent-4.5.22/api/src/main/java/org/semanticweb/owlapi/util/Construct.java)

Reply:
Yep, CINT is for concept intersection. There is an outstanding issue with that class, the weird naming might depend on it.

Reply:
If I tell people my ontology is written in a language called UCINETERIQ they will not use it 🤣